crates/ruff_python_formatter/resources/test/fixtures/ruff/expression/lambda.py

```diff
@@ -0,0 +1,24 @@
+# Leading
+lambda x: x  # Trailing
+# Trailing
+
+# Leading
+lambda x, y: x  # Trailing
+# Trailing
+
+# Leading
+lambda x, y: x, y  # Trailing
+# Trailing
+
+# Leading
+lambda x, /, y: x  # Trailing
+# Trailing
+
+# Leading
+lambda x: lambda y: lambda z: x  # Trailing
+# Trailing
+
+# Leading
+lambda x: lambda y: lambda z: (x, y, z)  # Trailing
+# Trailing
+
```

Comment:
Can we add some more examples where the parenthesized body expands over multiple lines?

Reply:
ed78487efe33648761550d9cf44ab08ee7c915d1